domehunter/__init__.py

```diff
@@ -86,70 +230,291 @@ def is_home(self):
     @property
     def status(self):
         """Return a text string describing dome rotators current status."""
-        pass
-
+        return self._dome_status
 
-##################################################################################################
+###############################################################################
 # Methods
-##################################################################################################
+###############################################################################
 
     """These map directly onto the AbstractMethods created by RPC."""
 
     def abort(self):
-        """Stop everything."""
+        """Stop everything by switching the dome motor on/off relay to off."""
+        # note, might want another way to do this incase the relay fails/sticks
+        # one way might be cut power to the automationHAT so the motor relays
+        # will receive no voltage even if the relay is in the open position?
+        self._stop_moving()
         pass
 
-    def getAzEl(self):
-        """Return AZ and Elevation."""
+    def getAz(self):
+        """Return current Azimuth of the Dome."""
+        if self.dome_az is None:
+            self.calibrate()
```

Comment:
```suggestion
            print("Cannot return Azimuth as Dome is not yet calibrated. Run calibration loop")
```
This is not good as it will automatically cause stuff to move, which might be dangerous in a `getAz` function that appears to just return info.

Let's make a rule here that passive calls for info should never move anything. The only things that should move things are functions/methods that clearly indicate they will move things.